mne/decoding/tests/test_receptive_field.py

```diff
@@ -176,6 +195,106 @@ def test_receptive_field():
     assert_raises(ValueError, rf.fit, X, y)
 
 
+def test_time_delaying_fast_calc():
+    """Test time delaying and fast calculations."""
+    X = np.array([[1, 2, 3], [5, 7, 11]]).T
+    # all negative
+    smin, smax = -2, -1
+    X_del = _delay_time_series(X, smin, smax, 1.)
+    X_del.shape = (X.shape[0], -1)
```

Comment:
Any chance you could add some comments here so that it'll be easier for somebody else to debug?